scalereg/reg6/models.py

```diff
@@ -34,7 +34,7 @@ class Order(models.Model):
   # basic info
   order_num = models.CharField(max_length=10, primary_key=True,
     help_text='Unique 10 upper-case letters + numbers code')
-  valid = models.BooleanField()
+  valid = models.BooleanField(default=False)
```

Comment:
I have all the BooleanFields modified in a local branch. Perhaps just drop this commit as well?